apps/condo/domains/meter/schema/Meter.js

```diff
@@ -163,6 +164,21 @@ const Meter = new GQLListSchema('Meter', {
             isRequired: true,
             defaultValue: false,
         },
+        meta: {
+            schemaDoc: 'Meter metadata. Can be used to store additional settings from integrations',
+            type: Json,
+            isRequired: false,
+        },
+        importRemoteSystem: {
```

Comment:
Why we need it here? We have `MeterReadingSource` schema to clarify, where readings are came from...